RoslynPluginGenerator/DataModel/PluginProperties.cs

```diff
@@ -16,21 +17,21 @@ namespace SonarQube.Plugins.Roslyn
     /// Describes a collection of Maven POM properties
     /// </summary>
     /// <remarks>This class is XML-serializable</remarks>
-    public class PluginProperties : StringDictionary, IXmlSerializable
+    public class PluginProperties : Dictionary<string, string>, IXmlSerializable
```

Comment:
It turns out that StringDictionary forces all of the keys to lower-case, which isn't what we want, so I switched to deriving from Dictionary<> instead. I've updated the unit tests to check the case of the keys is preserved.